README.md

```diff
@@ -1,143 +1,415 @@
+| ------------------------------- | ----------------- |
+| Attended our conference         | 5                 |
+| Spoke at a meetup               | 7                 |
+| Opened a pull request on GitHub | 4                 |
+| Answered a forum question       | 2                 |
+
+Then make another table that records the activities for each member. Capture the member's name, activity type, date, and activity score from the previous table.
+
+| Member Name       | Activity Type             | Date       | Score |
+| ----------------- | ------------------------- | ---------- | ----- |
+| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 5     |
+| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 4     |
+| Marie Monzo       | Answered a forum question | 2020-01-14 | 2     |
+| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 7     |
+
+Once we have a member's activity history, we look at both the _maximum_ and _sum_ of the activity scores. The **maximum** tells us the largest contribution the member has made, and the **sum** gives us a picture of their overall participation level.
+
+| Member Name       | Max Score | Total Score | Love (= Max Score) |
+| ----------------- | --------- | ----------- | ------------------ |
+| Jennifer Lewis    | 7         | 12          | 7                  |
+| Terrence Fletcher | 4         | 4           | 4                  |
+| Marie Monzo       | 2         | 2           | 2                  |
+
+It’s important to track both the sum and the max, but the **max** tells us more about the most impactful contribution the member can make, and helps determine which opportunity to offer next. More on that below.
+
+> "Jennifer's biggest contribution this year was organizing that meetup, an 8/10 for us. So in our database, her love is 8.”
+
+## Reach
+
+Reach is a measure of a community member’s influence, and takes into account a member's reputation and sphere of influence.
+
+> "Avery is one of our highest-reach community members. They're constantly engaging with folks on twitter and they run a blog with thousands of visitors per month. They're also a senior engineer at a well-respected agency."
+
+Unlike love, reach is independent of a particular community. Exceptions do apply for established communities where understanding a member's reach solely inside of that community is useful, but for smaller communities looking to grow it's generally more important to know about a member's reach in the broad sense.
+
+Why do we care about reach? Because individuals with high reach will attract others to join them as they journey through our solar system. Our relationship and love-building investment in high reach individuals may pay off bigger as a result. Everyone has some reach though, and one of our jobs as an orbit modeler is to help each of our members grow their sphere of influence. As we'll see in a minute, doing that also helps increase the gravity of our community.
```

Comment:
```suggestion
Why do we care about reach? Because **individuals with high reach will attract others to join them as they journey through our solar system.** Our relationship and love-building investment in high reach individuals may pay off bigger as a result. Everyone has some reach though, and one of our jobs as an orbit modeler is to help each of our members grow their sphere of influence. As we'll see in a minute, doing that also helps increase the gravity of our community.
```